src/index.js

```diff
@@ -225,7 +224,7 @@ export async function getVCData(credential, options = {}) {
   const formatter = new Intl.DateTimeFormat('en-US', { year: 'numeric', month: 'long', day: 'numeric' });
   const date = formatter.format(issuanceDate);
   const expirationDateStr = expirationDate ? formatter.format(expirationDate) : '';
-  const qrImage = generateQR && (await generateQRImage(credential, qrUrl));
+  const qrImage = generateQRImage && (await generateQRImage(credential));
```

Comment:
should we validate that `generateQRImage` is actually a function before trying to invoke it?

Reply:
yeah guess we can, theres quite a bit of cleanup this lib needs